client/components/telecom/telephony/alias/members/telecom-telephony-alias-members-add-modal.component.js

```diff
@@ -0,0 +1,13 @@
+angular.module('managerApp').controller('telecomTelephonyAliasMembersAddModal', class telecomTelephonyAliasMembersAddModal {
```

Comment:
No refactor is done, and will be made once the design/feature will be revisited

Reply:
Did you create a JIRA ticket ? 😄

Reply:
For the fix, the one linked in the description